pyproject.toml

```diff
@@ -42,7 +42,7 @@ dependencies = [
 [project.optional-dependencies]
 test = [
     "click",
-    "pytest",
+    "pytest<8.0.0",
```

Comment:
Looks like this needs to be rebased to `main`, since didn't this fix already go in?